src/analysis/AnalysisClient.js

```diff
@@ -37,12 +37,18 @@ class AnalysisClient {
   }
 
   _getBasicROIData(pairedProduct, amazonFBACost){
-    let totalPaidToWalmart = +this._getTotalAmountPaidToWalmart(pairedProduct.walmartProd).toFixed(2);
-    let totalCostPerItem = (totalPaidToWalmart + amazonFBACost);
-    let dollarROIPerItem = +(parseFloat(pairedProduct.amazonProd.price) - totalCostPerItem).toFixed(2);    
+    /*
+      to determine when to do comparison with amazonPrice or lowestOfferPrice
+      when doing Simple Cost analysis, the lowestOffer obj will be empty, so it will use amazon price
+      when doing secondary analysis, items with lowest offer will use lowestOffer price, other items will use amazonPrice
+    */
+    const amazonPrice = (Object.keys(pairedProduct.amazonProd.lowestOfferInfo).length === 0) ? pairedProduct.amazonProd.price : pairedProduct.amazonProd.lowestOfferInfo.lowestOfferInfo.Price.LandedPrice.Amount ;
+    const totalPaidToWalmart = +this._getTotalAmountPaidToWalmart(pairedProduct.walmartProd).toFixed(2);
+    const totalCostPerItem = (totalPaidToWalmart + amazonFBACost);
+    const dollarROIPerItem = +(parseFloat(amazonPrice) - totalCostPerItem).toFixed(2);    
```

Comment:
Similar comment here. Are we ever actually treating the `amazonPrice` as a string anywhere? If not, then it just store it as an int the first time so you don't have to do multiple conversions of it later. We should try to reduce the conversions as much as possible.